### PR TITLE
Typo Fix

### PR DESCRIPTION
**Pull request by mmvergara**
I got lazy sorry,

image

### CODE REVIEW

**Comment by realstealthninja** [comment]
Please do follow the pull request description given to you as it makes it easier for us to review

**Review by realstealthninja — DISMISSED** [review]
Nice spotting the typo!

**Review by Panquesito7 — APPROVED** [review]
Cool! Thanks. 🎉

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Looks like the whole file will need updating to our guidelines, as the CI is complaining about it.
Would you like to work on this, @mmvergara, or create an issue for the same?

**Comment by mmvergara** [comment]
> Looks like the whole file will need updating to our guidelines, as the CI is complaining about it. Would you like to work on this, @mmvergara, or create an issue for the same?

i'd like to work on it but sorry i don't really know how CI works, im kinda new to github. i might ruin something.

**Comment by realstealthninja** [comment]
> > Looks like the whole file will need updating to our guidelines, as the CI is complaining about it. Would you like to work on this, @mmvergara, or create an issue for the same?
> 
> i'd like to work on it but sorry i don't really know how CI works, im kinda new to github. i might ruin something.

no no he is asking you to fix the whole file and make it look better.
ie documenting code

**Comment by mmvergara** [comment]
What  did i just do lol, sorry i really don't know how the flow in these PR's work 💀

**Comment by realstealthninja** [comment]
you merged your pr with the main branch! i.e update your pull request so its up to date

File: greedy_algorithms/dijkstra.cpp [hunk 1]
```diff
@@ -11,13 +11,13 @@ class Graph {
 
     // Constructs a graph with V vertices and E edges
```
Comment (realstealthninja) [inline greedy_algorithms/dijkstra.cpp:12]:
```suggestion
    /**
     * @brief Constructs a graph
     * @param V number of vertices of the graph
     */
```

File: greedy_algorithms/dijkstra.cpp [hunk 2]
```diff
@@ -27,12 +27,13 @@ class Graph {
         this->vertexNum = V;
     }
 
-    // Adds the given edge to the graph
+    // Adds an edge to the graph
```
Comment (realstealthninja) [inline greedy_algorithms/dijkstra.cpp:30]:
```suggestion
    /** 
     * @brief Adds an edge to the graph
     * @param src the graph the edge should be added to
     * @param dst the position where the edge should be added to
     * @param weight the weight of the edge that should be added
     * @returns void
     */
```

**Review by realstealthninja — CHANGES_REQUESTED** [review]
when we mean `fix` the algorithm we mean is document and comment on the algorithm 
make it neat and understandable. 
try reading the contribution guidelines

**Comment by mmvergara** [comment]
> 1. Please use `climits` instead of `limits.h`.
> 2. Do not use `using namespace std`. Use `using <function>` (such as `std::cout`) or directly their namespace name.
> 3. Make sure to provide documentation and explanation of the algorithm, author, links, and **self-test implementations**.
> 
> If you need any help with this, let us know! 🙂

aight yeah. can someone actually finish this PR for me, sorry i'm not so used to github or C++ or this algorithm (yet).

**Comment by Panquesito7** [comment]
I'll work on this in the next few days. Thank you for contributing. 🙂

**Pull request closed without merging**